Master-OB-OpenAPI.json

```diff
@@ -14446,6 +14420,521 @@
         "items": {
           "$ref": "#/components/schemas/TaskResult"
         }
+      },
+      "ProgramName": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "Name of the Program.",
+            "x-ob-item-type": "StringItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "Use this field to store the name of the VPP Program that the site is enrolled in",
+            "x-ob-sample-value": {}
+          }
+        ]
+      },
+      "EligibleDeviceType": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "EligibleDeviceType",
+            "x-ob-item-type": "StringItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "Set this to Battery for VPP Contract",
+            "x-ob-sample-value": {}
+          }
+        ]
+      },
+      "MaxEventsPerSeason": {
```

Comment:
Should this be SeasonEventsMax?  In the word order pattern of general-specific-modifier?